src/common/snippets/tests/src/pass/gn_decomposition.cpp

```diff
@@ -44,22 +44,20 @@ TEST_P(GNDecompositionTest, GNDecomposition) {
 
 namespace {
 
-const std::vector<ov::PartialShape> input_shapes{
-    {1, 8},
-    {1, 8, 18},
-    {1, 16, 8, 5},
-    {3, 8, 2, 2, 3},
-    {3, 8, 2, 2, 3, 3}
-};
+const std::vector<ov::PartialShape> gn_decomposition_input_shapes{{1, 8},
```

Comment:
I'd like to understand the purpose of this change to avoid the same problems in the future.
Could you please elaborate a little bit? The previous `input_shapes` were in unnamed namespace. It should be safe for fast build (combining multiple translation units into a single one). Correct me please If I'm wrong

Reply:
It looks like is not safe when UNITY enabled as all goes into one translation unit. Maybe add some dedicated named test namespace will solve it and use same name will be also possible.

Reply:
Yeah, since these namespaces are unnamed, there is conflict in common translation unit.
We usually name the namespaces which contains tests instances:
https://github.com/openvinotoolkit/openvino/blob/260cf42126ee322f49f5f8afc5d189d8055e98ba/src/common/snippets/tests/src/pass/fuse_transpose_brgemm.cpp#L41

So as I understand, there are two options: to rename these variables with shapes, as you did, or name namespaces, as it's already implemented in other tests.

I'd suggest to leave it so and merge as it is.
However, there is one more test with `input_shapes` in unnamed namespace:
https://github.com/openvinotoolkit/openvino/blob/b068ea2a338b2c903fedd78342b9116939fb92c6/src/common/snippets/tests/src/pass/softmax_decomposition.cpp#L42-L43

May I ask you please to try to name namespaces (I hope it will work also) and fix softmax tests as well?

Thank you in advance!

Reply:
Done

Reply:
Thanks a lot!